integration-tests/data-dump-loading-tests/sakila_dump.sql

```diff
@@ -80,7 +80,7 @@ CREATE TABLE `address` (
   `city_id` smallint unsigned NOT NULL,
   `postal_code` varchar(10) DEFAULT NULL,
   `phone` varchar(20) NOT NULL,
-  `location` geometry NOT NULL /* UNSUPPORTED SYNTAX (https://github.com/dolthub/dolt/issues/3229): SRID 0 */,
+  `location` geometry NOT NULL /*!80003 SRID 0 */,
```

Comment:
Whoohoo! Awesome to get rid of one of the last remaining unsupported syntax comments in the Sakila dump file!